pkg/providers/imagefamily/bootstrap/aksbootstrap.go

```diff
@@ -583,6 +583,10 @@ func KubeletConfigToMap(kubeletConfig *KubeletConfiguration) map[string]string {
 	if kubeletConfig.ImageGCLowThresholdPercent != nil {
 		args["--image-gc-low-threshold"] = fmt.Sprintf("%d", ptr.Int32Value(kubeletConfig.ImageGCLowThresholdPercent))
 	}
+	// ShouldConfigureSwapFile is currently hardcoded to false, so can't support propagating this yet
```

Comment:
This comment doesn't seem to match the actual field it's talking about?
`ShouldConfigureSwapFile` vs `FailSwapOn`

Reply:
`ShouldConfigSwapFile` is a different internal setting in bootstrap. It - and possibly other settings like `SwapSizeMB` - would need to be coordinated to support proper control over swap file. For self-hosted, probably worth waiting for scriptless bootstrap to implement.